mlos_core/mlos_core/spaces/adapters/llamatune.py

```diff
@@ -89,8 +89,9 @@ def target_parameter_space(self) -> ConfigSpace.ConfigurationSpace:
 
     def inverse_transform(self, configurations: pd.DataFrame) -> pd.DataFrame:
         target_configurations = []
-        for (_, config) in configurations.iterrows():
-            configuration = ConfigSpace.Configuration(self.orig_parameter_space, values=config.to_dict())
+        for config in configurations.itertuples(index=False):
+            config = config._asdict()
+            configuration = ConfigSpace.Configuration(self.orig_parameter_space, values=config)
```

Comment:
`config` is used only once, so maybe
```suggestion
            configuration = ConfigSpace.Configuration(
                self.orig_parameter_space, values=config._asdict())
```